config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits.yaml

```diff
@@ -152,13 +170,16 @@ presubmits:
           privileged: true
     annotations:
       testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-azure
-      testgrid-tab-name: capi-pr-e2e
+      testgrid-tab-name: capz-capi-pr-e2e-main
```

Comment:
`capz-capi-pr-e2e-main` looks a bit awkward, what's the capz prefix needed for?

Reply:
you said we should have capz as a prefix for tab names similar to what cluster-api did

Reply:
changed it